roles/foreman_devel/tasks/main.yml

```diff
@@ -1,3 +1,24 @@
 ---
 - include_tasks: github_push_ssh.yml
   when: foreman_devel_github_push_ssh
+
+- include_tasks: check_forks_of_repos.yml
+  when: katello_devel_github_username is defined and foreman_installer_options_internal_use_only is defined
+
+- name: "Add OS hostname to hosts file" 
+  lineinfile:
+    dest: /etc/hosts
+    regexp: ".*{{ ansible_fqdn }}.*$"
+    line: "{{ hostvars[inventory_hostname]['ansible_default_ipv4']['address'] }} {{ ansible_fqdn }}"
+    state: present
+  become: true
+
+- include_tasks: rhel.yml
+  when: ansible_distribution == "RedHat"
+
+- name: "Install systemd-devel"
+  # This is required by journald-native gem
```

Comment:
@ekohl is this related to the structured logging that went in?

Reply:
Yes. I'd prefer this to be added to https://github.com/theforeman/puppet-katello_devel/blob/e52881cfccffa16de434e80bd2af4a2ad23b9b1b/manifests/install.pp#L4 so it all lives in one place.

Reply:
@ekohl  OK, no problem. Should I add it there in this pull request or you create a new pull requests for it?

Reply:
https://github.com/theforeman/puppet-katello_devel/pull/151